src/swap/letsexchange.js

```diff
@@ -0,0 +1,294 @@
+// @flow
+
+import { lt } from 'biggystring'
+import {
+  type EdgeCorePluginOptions,
+  type EdgeCurrencyWallet,
+  type EdgeSpendInfo,
+  type EdgeSwapInfo,
+  type EdgeSwapPlugin,
+  type EdgeSwapQuote,
+  type EdgeSwapRequest,
+  type EdgeTransaction,
+  SwapBelowLimitError,
+  SwapCurrencyError
+} from 'edge-core-js/types'
+
+import {
+  type InvalidCurrencyCodes,
+  checkInvalidCodes,
+  makeSwapPluginQuote, getCodes
+} from '../swap-helpers.js'
+
+const pluginId = 'letsexchange'
+const swapInfo: EdgeSwapInfo = {
+  pluginId,
+  displayName: 'LetsExchange',
+  supportEmail: 'support@letsexchange.io'
+}
+
+const orderUri = 'https://api.letsexchange.io/?exchangeId='
+const uri = 'https://api.letsexchange.io/api/v1/'
+
+const expirationMs = 1000 * 60 * 20
+
+type QuoteInfo = {
+  transaction_id: string,
+  status: string,
+  coin_from: string,
+  coin_to: string,
+  network_from: string,
+  network_to: string,
+  deposit_amount: string,
+  withdrawal_amount: string,
+  deposit: string,
+  deposit_extra_id: string,
+  withdrawal: string,
+  withdrawal_extra_id: string,
+  rate: string,
+  fee: string,
+  return: string,
+  return_extra_id: string,
+  final_amount: string,
+  hash_in: string,
+  hash_out: string,
+  isEstimate: boolean
+}
+
+const dontUseLegacy = {
+  DGB: true
+}
+
+const INVALID_CURRENCY_CODES: InvalidCurrencyCodes = {
+  from: {
+    ETH: ['MATIC'],
+    AVAX: 'allTokens',
+    FTM: 'allTokens',
+    MATIC: 'allCodes'
+  },
+  to: {
+    ETH: ['MATIC'],
+    AVAX: 'allTokens',
+    FTM: 'allTokens',
+    MATIC: 'allCodes',
+    ZEC: ['ZEC']
+  }
+}
+
+async function getAddress(wallet: EdgeCurrencyWallet, currencyCode: string) {
+  const addressInfo = await wallet.getReceiveAddress({ currencyCode })
+  return addressInfo.legacyAddress && !dontUseLegacy[currencyCode]
+    ? addressInfo.legacyAddress
+    : addressInfo.publicAddress
+}
+
+function networkCodeConverter(networkCode: string): string {
+  switch (networkCode) {
+    case 'ETH':
+      networkCode = 'ERC20';
+      break;
+    case 'BNB':
+      networkCode = 'BEP2';
+      break;
+    case 'TRX':
+      networkCode = 'TRC20';
+      break;
+    case 'BSC':
+      networkCode = 'BEP20';
+      break;
+    default:
+      break;
+  }
+
+  return networkCode;
+}
```

Comment:
Don't use a 'switch case' function, use the 'getCodesWithMainnetTranscription' instead.
Here is the link to the function: https://github.com/EdgeApp/edge-exchange-plugins/blob/80fac2c1ca153234f613241e4a5e7ef788c2a731/src/swap-helpers.js#L188
And this is an example usage: https://github.com/EdgeApp/edge-exchange-plugins/blob/80fac2c1ca153234f613241e4a5e7ef788c2a731/src/swap/changenow.js#L105

Reply:
Fixed it.